src/jwt/create-jwt-regex.js

```diff
@@ -55,4 +61,4 @@ const jwt = jsonwebtoken.sign(payload, privateKey, {
     keyid: keyId
 });
 
-console.log(`JWT string: ${jwt}`);
+console.log(`JWT: ${jwt}}`);
```

Comment:
I think theres a small typo adding two } if I'm not wrong

Reply:
Ouch yes fixed

Reply:
perfect